TrueOrFalse.java

```diff
@@ -3,14 +3,23 @@
 
 public class TrueOrFalse {
 
-	public static void main(String[] args) {
-		int number = 1;
-		if (number == 0) {
-			System.out.println("偽");
-		} else if (number == 1) {
+	public static void main(String[] args) throws Exception {
+		if (isBooleanNumber(1)) {
 			System.out.println("真");
+		} else {
+			System.out.println("偽");
 		}
 
 	}
 
+	public static Boolean isBooleanNumber(int number) throws Exception {
```

Comment:
いいですね！
このようにしてみるほうがいいかもしれません。
```suggestion
	public static boolean isTrueOrFalse(int number) throws Exception {
```